src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVmss.md

```diff
@@ -144,6 +158,67 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllocationMethod
+{{Fill AllocationMethod Description}}
```

Comment:
Please update these placeholders with actual descriptions of the parameter

Reply:
@markcowl this should be done by @twitchax